### ISSUE TITLE
Labels as an array of zeros when trying to load BBBC001 from MolNet

### ISSUE THREAD

**Issue by aaronrockmenezes** [deepchem/deepchem#3675]
## 🐛 Bug

When loading the labels for image set BBBC001 from MolNet, labels appear as a numpy array of zeros
Same goes for BBBC002.

## To Reproduce

import deepchem as dc
from deepchem.feat.molecule_featurizers import MolGraphConvFeaturizer
import numpy as np
import matplotlib.pyplot as plt
import pandas as pd

dataset_dc = dc.molnet.load_bbbc001()
tasks, dataset, transformers = dataset_dc
train, valid, test = dataset

x,y,w,ids = train.X, train.y, train.w, train.ids

print(y)

## Current output

[0. , 0. , 0. , 0.]

## Expected behavior

Get output as an array of the mean values of 2 manual counts of cells in each sample's labels .txt file

## Environment

* OS: Colab
* Python version: 3.10.12
* DeepChem version: 2.7.1
* RDKit version (optional):
* TensorFlow version (optional):
* PyTorch version (optional):
* Any other relevant information:
